lte/gateway/c/core/oai/test/spgw_task/test_spgw_procedures.cpp

```diff
@@ -175,12 +176,19 @@ class SPGWAppProcedureTest : public ::testing::Test {
       .qci = 1,
       .gbr = {.br_ul = 200000000, .br_dl = 100000000},
       .mbr = {.br_ul = 200000000, .br_dl = 100000000}};
+
+  void create_default_session(spgw_state_t* spgw_state, teid_t* ue_sgw_teid);
```

Comment:
We can just return `ue_sgw_teid` from this function call, instead of passing pointers around
```suggestion
teid_t create_default_session(spgw_state_t* spgw_state);
```

Reply:
I initially tried returning teid but this throws an error : error: void value not ignored as it ought to be ASSERT_EQ. If we want the function to return non-void value the  ASSERT_EQ  cannot be used in the function. Instead of ASSERT_EQ EXPECT_EQ should be used.  ASSERT_* macros cause termination of test case, EXPECT_* macros continue its evaluation.

Reply:
Let's use EXPECT_* macros as much as possible, since this is a function call which is mostly used to initialize the UE state before running the actual testing logic (such as dedicated bearer, etc). The expectation is that this should always pass. Further, in case of failure, having the test run fully will help expose multiple points of failures.